core/src/main/resources/context/applicationContext-core.xml

```diff
@@ -7,20 +7,20 @@
                            http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd
                            http://www.springframework.org/schema/jdbc http://www.springframework.org/schema/jdbc/spring-jdbc.xsd">
 
-    <context:component-scan base-package="com.es.core" />
+    <context:component-scan base-package="com.es.core"/>
 
     <bean id="dataSource" class="org.springframework.jdbc.datasource.DriverManagerDataSource">
-        <property name="driverClassName" value="org.h2.Driver" />
-        <property name="url" value="jdbc:h2:~/phoneshop/db/h2" />
-        <property name="username" value="sa" />
-        <property name="password" value="" />
+        <property name="driverClassName" value="${db.driver}"/>
+        <property name="url" value="${db.url}"/>
+        <property name="username" value="${db.user}"/>
+        <property name="password" value="${db.password}"/>
     </bean>
 
     <bean id="jdbcTemplate" class="org.springframework.jdbc.core.JdbcTemplate">
-        <property name="dataSource" ref="dataSource" />
+        <property name="dataSource" ref="dataSource"/>
     </bean>
 
-    <jdbc:initialize-database enabled="true" data-source="dataSource">
-        <jdbc:script location="classpath:db/schema.sql" />
+    <jdbc:initialize-database enabled="${db.insertSchema}">
```

Comment:
Currently, it cannot be configured via properties whether to import demo data or not for normal context. Please check the spring configuration under demodata module.

Reply:
Ooops... 
Fixed